File: library/src/main/java/com/github/javiersantos/materialstyleddialogs/UtilsAnimation.java
package com.github.javiersantos.materialstyleddialogs;

import android.content.Context;
import android.support.annotation.AnimRes;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;

class UtilsAnimation {

    static void startAnimation(Context context, ImageView image, @AnimRes int anim){
        Animation animation = AnimationUtils.loadAnimation(context,anim);
        image.startAnimation(animation);
    }
}

File: library/src/main/java/com/github/javiersantos/materialstyleddialogs/UtilsAnimation.kt
package com.github.javiersantos.materialstyleddialogs

import android.content.Context
import android.view.animation.AnimationUtils
import android.widget.ImageView
import androidx.annotation.AnimRes

internal object UtilsAnimation {
    @JvmStatic
    fun startAnimation(context: Context?, image: ImageView, @AnimRes anim: Int) =
        image.startAnimation(AnimationUtils.loadAnimation(context, anim))
}